hw/bsp/frdm-k82f/include/bsp/bsp.h

```diff
@@ -33,7 +33,9 @@ extern uint8_t __DATA_ROM;
 #define LED_RED_PIN         MCU_GPIO_PORTC(8)
 #define LED_GREEN_PIN       MCU_GPIO_PORTC(9)
 #define LED_BLUE_PIN        MCU_GPIO_PORTC(10)
+
 #define LED_BLINK_PIN       LED_RED_PIN
+#define LED_2               LED_GREEN_PIN
```

Comment:
What happened to LED_1? :frowning:

Reply:
That is not used anywhere afaik, I am not sure why BSPs export it but I'll do it here too.

Reply:
I switched to using the standard LED_n nomenclature, thanks for the suggestion.